src/declarative.js

```diff
@@ -134,15 +134,24 @@
       //    By disaling this module write-protection we gain performance.
       //    It could be useful to allow an option to enable or disable this.
       module.locked = true;
-      moduleObj[name] = value;
+
+      // export({name: value})
+      if (typeof name == 'object') {
+        for (var p in name)
```

Comment:
This API is defined to take instances of Object only from transpiler outputs and is independent of user code.